### PR TITLE
Added tooltip, moved tags

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/js/vue/utils/template-switch.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const templateVariables = {
     tagsExpanded: true,
   },
   GRIIDC: {
-    typeExpanded: false,
+    typeExpanded: true,
```
Comment (mickel1138) [inline assets/js/vue/utils/template-switch.js:23]: This can also be removed, as the default is true.

File: src/Entity/Dataset.php [hunk 2]
```diff
@@ -59,17 +59,17 @@ class Dataset extends Entity
     /**
      * Cold Storage Tag
      */
-    const TAG_COLDSTORAGE = 'Coldstorage';
+    const TAG_COLD_STORAGE = 'Cold Storage';
```
Comment (mickel1138) [inline src/Entity/Dataset.php:62]: Make public

File: assets/js/vue/utils/template-switch.js (hunk 1 above)
Comment (praneethpr) [reply assets/js/vue/utils/template-switch.js:23]: Okay

File: src/Entity/Dataset.php (hunk 2 above)
Comment (praneethpr) [reply src/Entity/Dataset.php:62]: We can run them when are merging this epic into develop?

File: src/Entity/Dataset.php [hunk 3]
```diff
@@ -81,9 +81,9 @@ class Dataset extends Entity
      * Valid Tags for a dataset.
      */
     const TAGS = [
```
Comment (fightingtexasaggie) [inline src/Entity/Dataset.php:83]: Tag hover values need to match the strings in the ticket.

**Review by fightingtexasaggie — COMMENTED** [review]
While it makes sense to put the hover text on the badges like was done here, I think the PO wanted them on the facet tags, instead of a repeat of the same text as hover.. It honestly makes sense in both places to me.

**Review by fightingtexasaggie — APPROVED** [review]

**Pull request merged**